exporters/variables-scss/src/generators/cssObjectGenerator.ts

```diff
@@ -1,51 +1,56 @@
-import { Token, TokenGroup } from '@supernovaio/sdk-exporters';
-import { toPlural } from '../helpers/stringHelper';
+import { Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
 import { tokenVariableName } from '../helpers/tokenHelper';
+import { toPlural } from '../helpers/stringHelper';
 
-export const generateObjectContent = (
-  tokens: Array<Token>,
-  tokenGroups: Array<TokenGroup>,
-  withParent: boolean,
-): string => {
-  return tokens.reduce((result, token) => {
-    const name = tokenVariableName(token, tokenGroups, withParent);
-    const numericPart = name.match(/\d+/)?.[0];
-    const prefix = `${token.origin?.name?.split('/')[0].toLowerCase()}-`;
-    const nonNumericPart = name.replace(prefix, '');
-
-    if (numericPart) {
-      result += `${numericPart}: $${name},\n`;
-    } else if (nonNumericPart) {
-      result += `${nonNumericPart}: $${name},\n`;
-    }
+export type CssObjectType = { [key: string]: string | object };
+
+const specialCases: { [key: string]: string } = {
+  'radius-full': 'full',
+};
+
+export const getNonNumericPart = (tokenName: string): string => {
```

Comment:
Well, this function does not do what I expected from the name. Based on the naming I would expect some parsing and checking the number in the string, but instead, it is just taking a name from the object and also lowercasing the string. That is far from the name and far from the single responsibility principle.

I would expect that lowercasing and handling special cases in token names would be two different steps/functions.

Reply:
refatored